CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Package `x/metrics` is added.
 - Tool `tools/metrics_exporter` binary is added.
 - `ProcessedTotal` and `FailedTotal` fields were added to `QueueInfo` struct.
+- The `asynq stats` command now supports a `--json` option, making its output a JSON object
```

Comment:
Instead of adding to the latest release, please add a new "Added" section under the Unreleased header.

```md
## [Unreleased]

### Added
- The `asynq stats` command now supports a `--json` option
```